[2024-06-04T17:11:11.278+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-04T17:11:11.297+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_weather.weather_api_ready_origin manual__2024-06-04T17:11:08.319038+00:00 [queued]>
[2024-06-04T17:11:11.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_weather.weather_api_ready_origin manual__2024-06-04T17:11:08.319038+00:00 [queued]>
[2024-06-04T17:11:11.308+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-04T17:11:11.323+0000] {taskinstance.py:2330} INFO - Executing <Task(HttpSensor): weather_api_ready_origin> on 2024-06-04 17:11:08.319038+00:00
[2024-06-04T17:11:11.330+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=295) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-04T17:11:11.332+0000] {standard_task_runner.py:63} INFO - Started process 297 to run task
[2024-06-04T17:11:11.333+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_weather', 'weather_api_ready_origin', 'manual__2024-06-04T17:11:08.319038+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/dag_ETL_weather.py', '--cfg-path', '/tmp/tmpjd406yuk']
[2024-06-04T17:11:11.334+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask weather_api_ready_origin
[2024-06-04T17:11:11.388+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_weather.weather_api_ready_origin manual__2024-06-04T17:11:08.319038+00:00 [running]> on host 6b740a15ab9e
[2024-06-04T17:11:11.705+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='dag_ETL_weather' AIRFLOW_CTX_TASK_ID='weather_api_ready_origin' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T17:11:08.319038+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T17:11:08.319038+00:00'
[2024-06-04T17:11:11.711+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-04T17:11:11.712+0000] {baseoperator.py:399} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2024-06-04T17:11:11.714+0000] {http.py:137} INFO - Poking: /data/2.5/weather?lat=-21.78&lon=-43.34&APPID=7a02546f64f1b21e9f2a742c236f852b
[2024-06-04T17:11:11.734+0000] {base.py:84} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-06-04T17:11:12.347+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-06-04T17:11:12.349+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-04T17:11:12.365+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_weather, task_id=weather_api_ready_origin, run_id=manual__2024-06-04T17:11:08.319038+00:00, execution_date=20240604T171108, start_date=20240604T171111, end_date=20240604T171112
[2024-06-04T17:11:12.434+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-04T17:11:12.461+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/models/baseoperator.py:1296: AirflowProviderDeprecationWarning: Call to deprecated class SimpleHttpOperator. (Class `SimpleHttpOperator` is deprecated and will be removed in a future release. Please use `HttpOperator` instead.)
  result = cls.__new__(cls)

[2024-06-04T17:11:12.529+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-04T17:11:12.532+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
